modules/sdk/project-templates/project-templates-soy-portlet/src/main/resources/archetype-resources/pom.xml

```diff
@@ -69,88 +69,71 @@
 				</executions>
 			</plugin>
 			<plugin>
-				<groupId>org.codehaus.mojo</groupId>
-				<artifactId>exec-maven-plugin</artifactId>
-				<version>1.5.0</version>
+				<groupId>com.liferay</groupId>
+				<artifactId>com.liferay.portal.tools.soy.builder</artifactId>
+				<version>@com.liferay.portal.tools.soy.builder.version@</version>
+
 				<executions>
 					<execution>
-						<id>transpile-js</id>
+                        <id>transpile-js</id>
 						<phase>process-classes</phase>
 						<goals>
-							<goal>exec</goal>
+							<goal>transpile-js</goal>
 						</goals>
-						<configuration>
-							<executable>${project.build.directory}/node/node</executable>
-							<workingDirectory>${project.build.outputDirectory}/META-INF/resources</workingDirectory>
-							<arguments>
-								<argument>${node.modules.dir}/metal-cli/index.js</argument>
-								<argument>build</argument>
-								<argument>--bundleFileName=</argument>
-								<argument>--dest</argument>
-								<argument>${project.build.outputDirectory}/META-INF/resources</argument>
-								<argument>--format</argument>
-								<argument>amd</argument>
-								<argument>--globalName=</argument>
-								<argument>--moduleName=</argument>
-								<argument>--soyDeps</argument>
-								<argument>${node.modules.dir}/lexicon*/src/**/*.soy</argument>
-								<argument>${node.modules.dir}/metal*/src/**/*.soy</argument>
-								<argument>--soyDest</argument>
-								<argument>${project.build.outputDirectory}/META-INF/resources</argument>
-								<argument>--soySrc</argument>
-								<argument>**/*.soy</argument>
-								<argument>--src</argument>
-								<argument>**/*.es.js</argument>
-								<argument>**/*.soy.js</argument>
-							</arguments>
-						</configuration>
+                        <configuration>
+                            <nodePath>${project.build.directory}/node/node</nodePath>
```

Comment:
lets add Defaults for the following variables and remove them from the pom

`bundleFileName`
`format`
`globalName`
`moduleName`
`soySrcs`
`soySrc`
`srcs`
`src`